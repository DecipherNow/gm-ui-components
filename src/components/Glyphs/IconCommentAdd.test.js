import React from "react";
import { shallow } from "enzyme";
import IconCommentAdd from "./IconCommentAdd.svg";
describe("IconCommentAdd", () => {
  it("matches snapshot", () => {
    
    const aIconCommentAdd = shallow(<IconCommentAdd />).find("IconCommentAdd.svg").dive();
    console.log(aIconCommentAdd.debug())
    expect(aIconCommentAdd).toMatchSnapshot();
  });
});