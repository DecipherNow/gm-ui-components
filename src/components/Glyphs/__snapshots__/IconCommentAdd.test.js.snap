// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconCommentAdd matches snapshot 1`] = `<IconMock />`;
